src/server/views/help/privacy-notice.html

```diff
@@ -0,0 +1,74 @@
+{% extends "layout.html" %}
+
+{% set pageTitle = "Submit a form to Defra privacy notice" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Submit a form to Defra privacy notice</h1>
```

Comment:
Mind replacing all the service names with `{{ serviceName }}` for consistency?

Thanks Dave

```suggestion
      <h1 class="govuk-heading-l">{{ serviceName }} privacy notice</h1>
```